extensions/amp-access/0.1/amp-access.js

```diff
@@ -333,6 +340,20 @@ export class AccessService {
   }
 
   /**
+   * @return {!Promise<string>}
+   * @private
+   */
+  getEncodedReaderId_() {
+    if (!this.encodedReaderIdPromise_) {
+      this.encodedReaderIdPromise_ = this.getReaderId_().then(readerId => {
+        dev.error(TAG, "bernie: readerId=", readerId);
```

Comment:
Use `dev.fine` instead. You can add `#log=3` to your URL to see all `dev.fine` messages.